carbonmark/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "carbonmark",
-  "version": "1.6.4",
+  "version": "1.6.5",
```

Comment:
why is this version incremented if there are no changes in `carbonmark`? or it's from lib/projectInfo?

Reply:
yeah it's because of the change to lib, any subgraphs that import that lib have to be incremented... annoying